console/src/app/pages/actions/add-action-dialog/add-action-dialog.component.ts

```diff
@@ -53,28 +106,28 @@ export class AddActionDialogComponent {
   public closeDialogWithSuccess(): void {
     if (this.id) {
       const req = new UpdateActionRequest();
-      req.setId(this.id);
-      req.setName(this.name);
-      req.setScript(this.script);
+      req.setId(this.form.value.id);
```

Comment:
```suggestion
      req.setId(this.id);
```

This won't work as this value is empty.
We have to set the id directly.

Reply:
Fixed, thanks a lot